### ISSUE TITLE
Could you release the training codes of VOC07 (ISD only) and VOC07 (ISD+CSD) in Supervised Learning in Table 1?

### ISSUE THREAD

**Issue by machengcheng2016** [soo89/ISD-SSD#4]
Greetings!
I am trying to reproduce the results of both VOC07 (ISD only) and VOC07 (ISD+CSD) in Supervised Learning in Table 1.
However, I meet the problem that the loss dramatically increases and the training process ruins. This problem may happens not only at first `1k` iterations, but also at `10k~20k` iterations, which is quite annoying!
So could you release the training codes of VOC07 (ISD only) and VOC07 (ISD+CSD) in Supervised Learning in Table 1? I wonder how you successfully get those results.
Thanks!

**Comment by soo89** [comment]
In my case, when we trained the model under a supervised learning setting, we changed the 'voc07_consistency.py' code.
change L101 from ' image_sets=[('2007', 'trainval'), ('2012', 'trainval')],' to 'image_sets=[('2007', 'trainval')],'.
